docs/src/prototypes/VirtualizedTable/itemsGenerator.ts

```diff
@@ -0,0 +1,52 @@
+import * as _ from 'lodash'
```

Comment:
can you please discuss with @layershifter if we need items generator at all? He did a experiment with virtualization with children API and said it worked good for him. For tree, generation is necessary as we are transforming tree structure to flat. For Table, I think we should create a list of pros and cons - or maybe even compare perf for these two options.

Reply:
Sure, will do that :)

Reply:
As agreed with @layershifter we would support only children API and not renderedItems prop